docs/dev/table/tableApi.md

```diff
@@ -2993,6 +2993,30 @@ MAP.at(ANY)
   </tbody>
 </table>
 
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 40%">Row functions</th>
+      <th class="text-center">Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+
+    <tr>
+      <td>
+        {% highlight java %}
```

Comment:
Please also add documentation for the Scala.

Reply:
seems like it is also missing `Map function` scala doc as well. adding in